android/app-newm/src/main/java/io/newm/NewmAppComposable.kt

```diff
@@ -188,6 +190,7 @@ internal fun NewmApp(
 internal fun NewmBottomNavigation(
     currentRootScreen: CircuitScreen?,
     eventLogger: NewmAppEventLogger,
+    showRecordStore: Boolean = false,
```

Comment:
Setting it by default to false until we find a good way to gate features for early preview.